### PR TITLE
Phoenix 1.5

### PR DESCRIPTION
**Pull request by CompuIves**
This is an attempt at upgrading Absinthe Phoenix to be compatible with the new version of Phoenix.PubSub. I've mostly followed the guide here: https://gist.github.com/chrismccord/e53e79ef8b34adf5d8122a47db44d22f, but I'm not sure if this is enough, the implementation using Phoenix.PubSub should probably change as well.

I figured I'd just open a PR to get the ball rolling!

### CODE REVIEW

File: lib/absinthe/phoenix/application.ex [hunk 1]
```diff
@@ -0,0 +1,11 @@
+defmodule Absinthe.Phoenix.Application do
```
Comment (CompuIves) [inline lib/absinthe/phoenix/application.ex:1]: I think this is probably only needed for running tests, since the pubsub of the application is reused?

**Comment by aj-foster** [comment]
Is there any interest in supporting both configuration keys, e.g. `[:pubsub_server]` or `[:pubsub][:name]` or `raise`? This could be friendly to folks who upgrade this package before upgrading Phoenix.

**Comment by maxguzenski** [comment]
I believe that is a mistake tight this project to phoenix config struct. Absinthe phoenix should has its own config. Something like that:

```elixir
config :absinthe_phoenix, 
    endpoint: MyWeb.Endpoint,
    pubsub: MyWeb.PubSub

# application.ex ->

# from 
supervisor(Absinthe.Subscription, [MyAppWeb.Endpoint])

# to 
supervisor(Absinthe.Subscription, [])
```

Is that test failed something expected or only happing with phoenix 1.5 ?

File: lib/absinthe/phoenix/application.ex (hunk 1 above)
Comment (IceDragon200) [reply lib/absinthe/phoenix/application.ex:1]: May I suggest using start_supervised in the phoenix_test?

```elixir
setup_all do 
  _pid = start_supervised!(Phoenix.PubSub.child_spec(name: Absinthe.Phoenix.PubSub))
end
```

**Comment by maxguzenski** [comment]
I'have hard trying to fix that failed test with no success. It works with phoenix 1.4 and pubsub 1.1... It looks like that is broadcasting correctly, but I cant assert it on tests. 

ps.: To start up PubSub you dont need a application.ex, you can put it on test_helper.ex

```elixir
#
# Code from Phoenix.PubSub test_helper.ex 
#
Logger.configure(level: :info)
Application.put_env(:phoenix_pubsub, :test_adapter, {Phoenix.PubSub.PG2, []})

Supervisor.start_link(
  [{Phoenix.PubSub, name: Absinthe.Phoenix.PubSub, pool_size: 1}],
  strategy: :one_for_one
)

ExUnit.start()

```

**Comment by benwilson512** [comment]
Working on this. We're gonna need an Absinthe.Phoenix 2.0

**Comment by aj-foster** [comment]
Have the goals of this PR been resolved by the 2.0 release?

**Comment by benwilson512** [comment]
Yup! Closing.

**Pull request closed without merging**